Kickstarter-iOS/DataSources/ProjectPageViewControllerDataSourceTests.swift

```diff
@@ -0,0 +1,487 @@
+@testable import Kickstarter_Framework
```

Comment:
Interesting theme in this file is that the data source will evaluate any of the cases in the enum as an existing section but obviously the number of rows will be zero unless it is set. Made testing a bit verbose but does explain the gist of what's happening behind the hood.

Reply:
Ah I see. When we were working on comment replies there was an explicit call to `padValuesForSection` which was required because we ran into some out of bounds errors. Never actually looked into the implementation details for the `set` method but it makes sense that it's in there. Thanks for digging into that!